scope_extensions-explainer.md

```diff
@@ -23,40 +23,60 @@ origins.
 Web app scope (defined by the `scope` field) is currently used for:
 1. Determining whether an app window's root document has left the app's scope
    (possibly invoking window UI informing the user of this).
-1. Constraining URLs appearing in manifest members like `start_url`, `file_handlers`, or `share_target`.
+1. Constraining URLs appearing in manifest members like `start_url`,
+   `file_handlers`, or `share_target`.
 
 The `scope_extensions` mechanism can expand all these behaviours to include
 other origins given agreement between the web app's primary origin and the
 associated origins.
 
 
 ## Proposal
+To extend the app scope, a developer must modify the web app manifest and host
+one or more association files.
 
-1. Add a `scope_extensions` member to the web app manifest specifying a list of
-   origin patterns to associate with.
+### Web app manifest
+Add a `scope_extensions` member to the web app manifest specifying a list of
+origins to include in the extended app scope. 
 
-   Example manifest located at `https://example.com/manifest.webmanifest`:
+Example manifest located at `https://example.com/manifest.webmanifest`:
    ```json
    {
      "id": "/",
      "name": "Example",
      "display": "standalone",
      "start_url": "/index.html",
+     "scope": "/app",
      "scope_extensions": [
-       {"origin": "*.example.com"},
-       {"origin": "example.co.uk"},
-       {"origin": "*.example.co.uk"},
-       {"origin": "*.example.co.uk"}
+       {"type": "registrable_domain", "value": "https://example.com"},
+       {"type": "origin", "value": "https://example.co.uk"},
```

Comment:
otherwise I have the question - does the registrable domain imply the origin? Or does it not imply the origin? So removing it removes that question. And this can just be an origin example.

Reply:
After discussion, we decided to have the registrable domain imply inclusion of the origin.  Removed .co.uk origin and added help-center origin example. Updated text below.